test/features/step_definitions/sdk_steps.rb

```diff
@@ -41,22 +46,43 @@
 
 Then(/^I should reach the complete step$/) do
   steps %Q{
-    Then page_title () should contain "Verification complete"
-    Then I should not see "back"
+    Then page_title () should contain "#{i18n.translate('complete.message')}"
+    Then page should not have back button
   }
 end
 
-Then(/^I can navigate back to the previous page with title "([^"]*)"$/) do | title |
+Then(/^I can navigate back to the previous page with title "([^"]*)"$/) do | key |
+  title = i18n.translate(key)
   steps %Q{
     When I click on back ()
     Then page_title () should contain "#{title}"
   }
 end
 
 When(/^I upload my document and selfie$/) do
+  selfie = i18n.translate('capture.face.upload_title')
   steps %Q{
     When I try to upload passport
-    Then page_title () should contain "Selfie"
+    Then page_title () should contain "#{selfie}"
     When I try to upload one_face
   }
 end
+
+Then(/^(.*) should include "([^"]*)"$/) do | page_element, key|
```

Comment:
consider making it explicit that this is doing translation? e.g. `should include translation ...`